bootstrap/local/variables.tf

```diff
@@ -0,0 +1,34 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+variable "id" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+variable "location" {
+  type = string
+}
+
+variable "tf_in_automation" {
+  type = bool
+}
+
+variable "suffix" {
+  type    = string
+  default = ""
```

Comment:
~why allow this?~
maybe a comment that this is not actually the default

Reply:
You're right, `suffix_override` would be more obvious